lib/miq_automation_engine/service_models/miq_ae_service_host.rb

```diff
@@ -21,6 +21,9 @@ class MiqAeServiceHost < MiqAeServiceModelBase
     expose :domain
     expose :files,                 :association => true
     expose :directories,           :association => true
+    expose :raw_set_node_maintenance
+    expose :raw_unset_node_maintenance
+    expose :raw_get_node_maintenance
```

Comment:
I'm worried about exposing methods called `raw_` to the automate model just for the look and feel, really.

I suggest one of two things:

1. change the lines here to 
```ruby
    expose :set_node_maintenance :method => :raw_set_node_maintenance
```
2. add wrapper methods (dropping the `raw`) to the `Operations` module that just call the underlying `raw` methods, and expose the wrapper methods here

I'd say for now to just go with option 1 because the wrappers don't add anything at the moment.